x-pack/plugins/screenshotting/server/screenshots/index.ts

```diff
@@ -8,7 +8,6 @@
 import type { CloudSetup } from '@kbn/cloud-plugin/server';
 import type { HttpServiceSetup, KibanaRequest, Logger, PackageInfo } from '@kbn/core/server';
 import type { ExpressionAstExpression } from '@kbn/expressions-plugin/common';
-import type { Optional } from '@kbn/utility-types';
```

Comment:
Before merging, we should undo all the changes under the `x-pack/plugins/screenshotting`